server/lib/handlebars.js

```diff
@@ -125,7 +125,7 @@ handlebars.registerHelper('currency', (value, props) => {
     return value.toLocaleString(locale, {
       style: 'currency',
       currency,
-      minimumFractionDigits: precision || 0,
+      minimumFractionDigits: precision || 2,
```

Comment:
Let's not impact the currencies without decimals

```suggestion
      minimumFractionDigits: precision || getDefaultCurrencyPrecision(currency),
```

Reply:
Thanks for pointing it out. Just corrected it. 👍🏽